pinot-plugins/pinot-input-format/pinot-csv/src/test/java/org/apache/pinot/plugin/inputformat/csv/CSVRecordReaderTest.java

```diff
@@ -324,6 +324,47 @@ public void testLineIteratorReadingDataFileWithUnparseableLines()
     Assert.assertEquals(1, genericRows.size());
   }
 
+  @Test
+  public void testLineIteratorReadingDataFileWithUnparseableLinesWithRewind()
+      throws URISyntaxException, IOException {
+    URI uri = ClassLoader.getSystemResource("dataFileWithUnparseableLines2.csv").toURI();
+    File dataFile = new File(uri);
+
+    CSVRecordReaderConfig readerConfig = new CSVRecordReaderConfig();
+    readerConfig.setSkipUnParseableLines(true);
+    final List<GenericRow> genericRows1 = readCSVRecords(dataFile, readerConfig, null, false);
```

Comment:
@swaminathanmanish 
I was wrong about this. It doesn't `rewind()`. And it doesn't matter because `readCSVRecords()` does an `init()` before reading. That's why the test is passing.